app/containers/LoginServices/Service.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { Text } from 'react-native';
+
+import { useTheme } from '../../theme';
+import I18n from '../../i18n';
+import { TIconsName } from '../CustomIcon';
+import { IItemService, IOauthProvider } from './interfaces';
+import styles from './styles';
+import * as ServiceLogin from './serviceLogin';
+import ButtonService from './ButtonService';
+
+const Service = React.memo(
+	({
+		CAS_enabled,
+		CAS_login_url,
+		Gitlab_URL,
+		server,
+		service
+	}: {
+		service: IItemService;
+		server: string;
+		Gitlab_URL: string;
+		CAS_enabled: boolean;
+		CAS_login_url: string;
+		storiesTestOnPress?: () => void;
+	}) => {
+		const { colors } = useTheme();
+
+		const { name } = service;
+		let modifiedName = name === 'meteor-developer' ? 'meteor' : name;
+		const icon = `${modifiedName}-monochromatic` as TIconsName;
+		const isSaml = service.service === 'saml';
+		let onPress: any = () => {};
```

Comment:
Will this value be changed?
Why is it not recommended to use let inside components, as you have no reference to which value that is, which causes some re-rendering.
Same buttonText